purism/librem/5r4/u-boot/default.nix

```diff
@@ -0,0 +1,67 @@
+{ stdenv, buildUBoot, fetchurl, fetchFromGitLab, lib, flex, bison }:
+let
+  firmware-imx = stdenv.mkDerivation (fa: {
+    pname = "firmware-imx";
+    version = "8.12";
+    src = fetchurl {
+      url =
+        "https://www.nxp.com/lgfiles/NMG/MAD/YOCTO/${fa.pname}-${fa.version}.bin";
+      sha256 = "1vr2wgjac718hp48arhdvxd7gib93zhdrbrla8w3xigc6szlfrvb";
+    };
+    unpackPhase = ''
+      cp $src firmware
+      chmod +x firmware
+      ./firmware --auto-accept
```

Comment:
This belongs into buildPhase or configurePhase

Reply:
It's a self-unpacking archive, so I would count this as unpacking